apps/site/app/[locale]/not-found.tsx

```diff
@@ -12,7 +12,7 @@ const NotFoundPage: FC = () => {
   const t = useTranslations();
 
   return (
-    <GlowingBackdropLayout>
+    <GlowingBackdropLayout kind="default">
```

Comment:
Does Error page also has this component?

Reply:
Nice catch, added it to the Error page 👍